### DISCUSSION TITLE
SenseVoiceLarge release updates?

### DISCUSSION THREAD

**Discussion by as4193** [modelscope/FunASR#1910] [Q&A] [answered]
I've been looking at SenseVoiceLarge benchmarks and have been impressed by how its outperforming other models. As such, I'm really excited about its potential applications in the field of ASR. But I noticed that only the Small model is available. Does anyone have any updates on when SensevVoiceLarge is expected to be released ? I'm eager to start exploring testing out and would love to hear any information you might have.

**Comment by LauraGPT** [comment, marked as answer]
Update as of **2026-07-15**: an official **SenseVoiceLarge checkpoint has not been publicly released, and there is no published release ETA** that we can reliably provide.

I rechecked the current official release surfaces:

- The [SenseVoice model zoo](https://github.com/FunAudioLLM/SenseVoice/blob/fbf91f3baccebec554dcee68708b0cda61d42805/README.md#model-zoo) links only `SenseVoiceSmall` on ModelScope and Hugging Face.
- The `FunAudioLLM` Hugging Face organization currently exposes `SenseVoiceSmall` and its GGUF conversion, but no Large checkpoint.
- A complete listing of the 1,065 public models under ModelScope's official `iic` group contains `SenseVoiceSmall` and `SenseVoiceSmall-onnx`, but no Large checkpoint.
- The repository releases contain the SenseVoice v1.0.0 release and llama.cpp runtime releases, but no SenseVoiceLarge weights.

The SenseVoiceLarge figures in the paper/benchmark describe the research model used for those experiments; they should not be interpreted as a downloadable checkpoint or a release-date commitment.

For work that can start now:

- Use [`iic/SenseVoiceSmall`](https://www.modelscope.cn/models/iic/SenseVoiceSmall) or [`FunAudioLLM/SenseVoiceSmall`](https://huggingface.co/FunAudioLLM/SenseVoiceSmall) for the released five-language ASR, language ID, emotion, and audio-event path.
- For CPU/edge deployment, use [`FunAudioLLM/SenseVoiceSmall-GGUF`](https://huggingface.co/FunAudioLLM/SenseVoiceSmall-GGUF).
- If the main requirement is higher-capacity ASR rather than SenseVoice's emotion/event outputs, evaluate the separate [`Fun-ASR-Nano-2512`](https://huggingface.co/FunAudioLLM/Fun-ASR-Nano-2512) family. It is not a drop-in SenseVoiceLarge replacement and has a different capability contract.

Please treat the official model zoo and repository releases as the canonical sources. Until a maintainer publishes a checkpoint and model card there, any date would be speculation.
